src/api/Dues/index.ts

```diff
@@ -0,0 +1,6 @@
+import { accessClient } from 'api';
+import { MemberShipDues } from 'model/dues/allDues';
+
+export const getAllDues = async (year: number, track?: string) => {
+  return await accessClient.get<MemberShipDues>(`/dues?year=${year}&track=${track}`);
```

Comment:
fetcher가 await한 결과를 반환할 필요는 없어보여서,
결국 그냥 프라미스를 반환한 것과 동일하게 동작할 것 같네용
```suggestion
export const getAllDues = (year: number, track?: string) => {
  return accessClient.get<MemberShipDues>(`/dues?year=${year}&track=${track}`);
```

Reply:
앗 그렇네요 수정했습니다! cb5840c8ae7ba4510b3833429098f2660d2fd673